web/docs/auth/auth-hooks.md

```diff
@@ -520,6 +528,54 @@ export const onAfterLogin: OnAfterLoginHook = async ({
 
 Read more about the data the `onAfterLogin` hook receives in the [API Reference](#the-onafterlogin-hook).
 
+### Refreshing the OAuth access token
+
+If you are using OAuth, you can use the `onAfterSignup` or `onAfterLogin` hooks to refresh the OAuth access token if the provider supports it.
+
+You'll need to import the OAuth client from the `wasp/server/oauth` module for the provider you are using. The OAuth client has a `refreshAccessToken` method that you can use to refresh the access token.
+
+Here is an example of how you can refresh the Google OAuth access token:
+
+<Tabs groupId="js-ts">
+
+<TabItem value="js" label="JavaScript">
+
+```js title="src/auth/hooks.js"
+import { google } from 'wasp/server/oauth'
+
+export const onAfterLogin = async ({ oauth }) => {
+  if (oauth.provider === 'google' && oauth.tokens.refreshToken !== null) {
+    const newTokens = await google.oAuthClient.refreshAccessToken(
+      oauth.tokens.refreshToken
+    )
+    log('new tokens', newTokens)
+  }
+}
+```
+</TabItem>
+
+<TabItem value="ts" label="TypeScript">
+
+```ts title="src/auth/hooks.ts"
+import type { OnAfterLoginHook } from 'wasp/server/auth'
+import { google } from 'wasp/server/oauth'
+
+export const onAfterLogin: OnAfterLoginHook = async ({ oauth }) => {
+  if (oauth.provider === 'google' && oauth.tokens.refreshToken !== null) {
+    const newTokens = await google.oAuthClient.refreshAccessToken(
+      oauth.tokens.refreshToken
+    )
+    log('new tokens', newTokens)
+  }
+}
+```
+</TabItem>
+</Tabs>
+
+Google provides you with the `accessTokenExpiresAt` field in the `oauth.tokens` object. You can use this field to determine when the access token expires and refresh it before it expires.
```

Comment:
```suggestion
Google exposes the `accessTokenExpiresAt` field in the `oauth.tokens` object. You can use this field to determine when the access token expires.
```
Subjective, but I like it because it's shorter.

I removed the "and refresh it before it expires." part because it suggests that they can use the field to refresh the token, which I don't think is right.